package.json

```diff
@@ -13,7 +13,7 @@
     "prettier:fix": "prettier '**/*.{js,ts,tsx,json}' --write",
     "check-code": "yarn tsc:check && yarn eslint:check && yarn prettier:check",
     "codegen": "apollo client:codegen --endpoint http://127.0.0.1:4000/graphql --target typescript --globalTypesFile=app/__generated__/globalTypes.ts -c ./apollo.client.js --watch",
-    "test": "LOGLEVEL=warn jest --runInBand",
+    "test": "LOGLEVEL=warn jest --runInBand --forceExit",
```

Comment:
did you check with --detectOpenHandles before use force exit? is better detect and fix the open handles